contracts/Pool.sol

```diff
@@ -479,8 +480,8 @@ contract Pool is PoolFDT {
     /**
         @dev   `from` and `to` should always be equal in this implementation.
         @dev   This means that the custodian can only decrease their own allowance and unlock funds for the original owner.
-        @param from   Address which holds to Pool FDTs.
-        @param to     Address which going to be the new owner of the `amount` FDTs.
+        @param from   Address which holds the Pool FDTs.
+        @param to     Address which will be the new owner of the `amount` of FDTs.
         @param amount Number of FDTs transferred.
      */
     function transferByCustodian(address from, address to, uint256 amount) external {
```

Comment:
So we should emit this 
`emit CustodyAllowanceChanged(from, msg.sender, oldAllowance, newAllowance);` So first parameter should be `tokenHolder` and that is `from/to` in this case while `msg.sender` here is custodian so it should be the second argument.